centos/Dockerfile

```diff
@@ -3,6 +3,18 @@ LABEL maintainer="Kong Core Team <team-core@konghq.com>"
 
 ENV KONG_VERSION 0.15.0
 
+ARG SU_EXEC_VERSION=0.2
+ARG SU_EXEC_URL="https://github.com/ncopa/su-exec/archive/v${SU_EXEC_VERSION}.tar.gz"
+
+RUN yum install -y -q gcc make \
+&& curl -sL "${SU_EXEC_URL}" | tar -C /tmp -zxf - \ 
+ && make -C "/tmp/su-exec-${SU_EXEC_VERSION}" \
+&& cp "/tmp/su-exec-${SU_EXEC_VERSION}/su-exec" /usr/bin \
+&& rm -fr "/tmp/su-exec-${SU_EXEC_VERSION}" \
+&& yum autoremove -y -q gcc make \
+&& yum clean all -q \
+&& rm -fr /var/cache/yum/* /tmp/yum_save*.yumtx /root/.pki
+
```

Comment:
I can confirm that this does solve the original problem and also runs nginx master process as PID 1.

Regarding, building su-exec from source, should we build it once and host it on bintray/s3 and use it? Or, we could use [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) to build su-exec and copy it in the production image.

Reply:
So far as building the rpm / yum asset there's these:
https://github.com/ncopa/su-exec/pull/4
https://bugzilla.redhat.com/show_bug.cgi?id=1393044

I think I'd be more inclined to make our own docker base image

Reply:
Not a blocker but only sharing thoughts/practices.